programs/bpf/tests/programs.rs

```diff
@@ -295,31 +306,35 @@ fn process_transaction_and_record_inner(
     let signature = tx.signatures.get(0).unwrap().clone();
     let txs = vec![tx];
     let tx_batch = bank.prepare_batch_for_tests(txs);
-    let (mut results, _, mut inner_instructions, _transaction_logs) = bank
+    let mut results = bank
         .load_execute_and_commit_transactions(
             &tx_batch,
             MAX_PROCESSING_AGE,
             false,
             true,
             false,
             &mut ExecuteTimings::default(),
-        );
+        )
+        .0;
     let result = results
         .fee_collection_results
         .swap_remove(0)
         .and_then(|_| bank.get_signature_status(&signature).unwrap());
-    (
-        result,
-        inner_instructions
-            .swap_remove(0)
-            .expect("cpi recording should be enabled"),
-    )
+    let inner_instructions = results
+        .execution_results
+        .swap_remove(0)
+        .details()
+        .expect("tx should be executed")
+        .clone()
```

Comment:
Can we get rid of this `.clone()` or at least move it further out?

Reply:
For what purpose, this is just a test

Reply:
You are right, sorry.